mubench.pipeline/data/detector_execution.py

```diff
@@ -125,7 +119,7 @@ def _run_mode_detector_argument(self):
     def _get_findings_path(self):
         raise NotImplementedError
 
-    def _get_detector_arguments(self, project_compile: ProjectCompile):
+    def _get_command(self, project_compile: ProjectCompile):
```

Comment:
Why do we need this? I don't see an invocation. This has been moved to the interface, right?